zebra-state/src/service/finalized_state/disk_format/upgrade.rs

```diff
@@ -275,16 +276,55 @@ impl DbFormatChange {
         // Start of a database upgrade task.
 
         let version_for_pruning_trees =
-            Version::parse("25.1.1").expect("Hardcoded version string should be valid.");
+            Version::parse("25.1.2").expect("Hardcoded version string should be valid.");
```

Comment:
If this merges after the subtree indexes (and it probably should), update this to a minor version bump